CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 #### Breaking
```

Comment:
We could make this non-breaking by implicitly treating anything without `/v*/` prefix as `/v1/`. I like not doing so, but just pointing out that we necessarily don't have to break compatibiltiy here.

Reply:
It is more of a question to the @Voxelot . I thought the idea is to break it now to require all SDK handle different version instead of blindly using `/graphql`.

Reply:
Im fine with this option as well if we want to limit churn.